lib/dor/text_extraction/ocr.rb

```diff
@@ -8,7 +8,7 @@ class Ocr
 
       def initialize(params = {})
         @cocina_object = params[:cocina_object]
-        @workflow_context = params[:workflow_context]
+        @workflow_context = params[:workflow_context] || {}
```

Comment:
this basically makes workflow_context optional by defaulting to an empty hash since some consumers of this class won't need to pass it in (e.g. just getting folder paths or cleaning up)